cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json

```diff
@@ -1749,22 +1614,7 @@
         "Pass/Fail Criteria": "Self Checking Test",
         "Test Type": "Directed Self-Checking",
         "Coverage Method": "Testcase",
-        "Link to Coverage": "",
-        "Review (Marton)": "?",
-        "Review (Robin)": "?",
-        "Review (Henrik)": "?"
-    },
-    {
-        "Reference document": "",
-        "Requirement Location": "",
-        "Feature": "Instruction boundaries",
-        "Sub Feature": "",
-        "Feature Description": "",
-        "Verification Goal": "While single stepping, execute misaligned loads/stores, push/pops, and table jumps, cause an external debug request while the instruction has started its sub operations, check that the external debug request does not interrupt the instructions.",
-        "Pass/Fail Criteria": "Check against ISS",
-        "Test Type": "Constrained-Random",
-        "Coverage Method": "Functional Coverage",
-        "Link to Coverage": "",
+        "Link to Coverage": "TODO: not covered",
```

Comment:
What do we do about this? We have tasks on it?

Reply:
Yes, there is a task on this